base/applications/taskmgr/taskmgr.h

```diff
@@ -15,7 +15,7 @@
 
 #define STATUS_WINDOW	2001
 #define STATUS_SIZE1	85
-#define STATUS_SIZE2	190
+#define STATUS_SIZE2	157 // he-IL.rc determines 72 requirement
```

Comment:
Just simply by spelling out in the comment why "157", based on your explanation given in the PR:
```suggestion
#define STATUS_SIZE2	157 // he-IL.rc determines minimum width:  72 == 157 - 85
```